circle.yml

```diff
@@ -24,6 +24,8 @@ dependencies:
     - sed -i "s/xxxxx/$(parity --keys-path ./keys --password ./parityPassword account new)/g" ./parity-genesis.json
     - sed -i "s/yyyyy/$(parity --keys-path ./keys --password ./parityPassword account new)/g" ./parity-genesis.json
     - sed -i "s/zzzzz/$(parity --keys-path ./keys --password ./parityPassword account new)/g" ./parity-genesis.json
+    # Compile contracts so artifacts are present for eslint
+    - ./node_modules/truffle/build/cli.bundled.js compile
```

Comment:
I don't understand this 2 ways: 1) the way `truffle compile` is called 2) why do we need to lint contract artefacts

Reply:
1. So you would be happier to compile with `./node_modules/.bin/truffle`?
2. The mining client JS files `require` some contract artifacts, and ESLint checks that all `require`d paths exist, and so without compiling them the linting check fails.